platform/darwin/test/MGLExpressionTests.mm

```diff
@@ -296,7 +296,7 @@ - (void)testConstantValueExpressionObject {
         float blue = (213.0 * alpha) / 255;
         MGLColor *color = [MGLColor mgl_colorWithColor:{ red, green, blue, alpha }]; // papayawhip
         NSExpression *expression = [NSExpression expressionForConstantValue:color];
-        NSArray *jsonExpression = @[@"rgba", @127.5, @119.5, @106.5, @0.5];
+        NSArray *jsonExpression = @[@"rgba", @255.0, @239.0, @213.0, @0.5];
         XCTAssertEqualObjects(expression.mgl_jsonExpressionObject, jsonExpression);
         XCTAssertEqualObjects([expression expressionValueWithObject:nil context:nil], color);
```

Comment:
This tests that Apple’s NSExpression code can get a constant value out of a constant value expression, but it doesn’t test that a JSON `rgba` expression gets converted to the right NSColor or UIColor. Can we add a test to that effect? I’m concerned that we’re only fixing the bug in one direction but not the other.

Reply:
Just added a test that hopefully covers your concern!